build/dsc.rake

```diff
@@ -57,8 +57,8 @@ eod
         puts "Downloading and Importing #{item_name}"
         cmd = "git clone #{dsc_repo} #{dsc_resources_path_tmp} && " +
           "cd #{dsc_resources_path_tmp} && "
-        cmd += "git checkout #{ENV['DSC_REF']} &&" if ENV['DSC_REF']
-        cmd += "git submodule update --init --recursive"
+        cmd += "git checkout #{ENV['DSC_REF']} && " if ENV['DSC_REF']
+        cmd += "git submodule update --init"
```

Comment:
Correct - OK for now given that didn't have a bearing on the generated types.

We should verify for sure though that we didn't exclude anything needed at runtime that wouldn't show up as different in type generation.